apps/table.html

```diff
@@ -372,7 +372,9 @@ <h5 class="modal-title" id="deleteModalLabel">Delete confirmation</h5>
 							else if (!d[key]) rs.push('')
 							else rs.push(d[key])
 						});
-						const btn = `<div id='open-delete'> <button class="btn btn-success" data-id='${rs[0]}' onclick='openView(this)'>Open</button> <button type='button' class='btn btn-danger DelButton' id='deleteBtn' data-id='${rs[0]}' data-name='${rs[1]}' onclick='deleteSld(this)' data-toggle='modal'> <i class='fas fa-trash-alt' ></i> </button></div>`
+						const btn = `<div id='open-delete'> <button class="btn btn-success" data-id='${rs[0]}' onclick='openView(this)'>Open</button> 
+							<button type='button' class='btn btn-info DownloadButton' id='downloadBtn' data-id='${rs[0]}' onclick='downloadSlide(this)' > <i class='fas fa-download' ></i> </button>
+							<button type='button' class='btn btn-danger DelButton' id='deleteBtn' data-id='${rs[0]}' data-name='${rs[1]}' onclick='deleteSld(this)' data-toggle='modal'> <i class='fas fa-trash-alt' ></i> </button></div>`
```

Comment:
@akhil-rana , can you do one more favour ? 
Please add this 'display:none' style to the delete button initially, so as we have already been calling checkUserPermissions() in line 379 of table.html , which will automatically change the 'display' to 'block', iff the user has apt. permissions. 
Currently, the delete button is being displayed to all the users, irrespective of whether they have permission or not. 
This change will help fix this bug, in this PR itself -
```suggestion
							<button type='button' class='btn btn-danger DelButton' style='display:none;' id='deleteBtn' data-id='${rs[0]}' data-name='${rs[1]}' onclick='deleteSld(this)' data-toggle='modal'> <i class='fas fa-trash-alt' ></i> </button></div>`
```

Reply:
It is already set to `none` in `table.css` here [table.css](https://github.com/camicroscope/caMicroscope/blob/develop/apps/table.css#L51)

Reply:
Oh! my bad.